### ISSUE TITLE
ZOMB: Pathfinding issue on zomb_underground

### ISSUE THREAD

**Issue by TeeSlayer** [LordSk/teeworlds#6]
Map: https://lolinet.info/download/maps/zomb_underground.map

See screenshots:

![alt zomb_screenshot_1](https://lolinet.info/download/zomb_screenshot_1.png)
![alt zomb_screenshot_2](https://lolinet.info/download/zomb_screenshot_2.png)

The boomer in the first screenshot can still proceed because the other zombies will push him over the edge in a few seconds, but the other one is the last surviving zombie and thus the players need to search the whole map in order to find and kill him.

**Comment by LordSk** [comment]
Yes I am aware of the issue, I am working on a fix. Thanks for reporting.

**Comment by LordSk** [comment]
This should be somewhat fixed, it may need more testing to be sure.
